src/main/java/net/atos/client/or/shared/exception/FoutExceptionMapper.java

```diff
@@ -19,7 +19,7 @@ public class FoutExceptionMapper implements ResponseExceptionMapper<FoutExceptio
 
     @Override
     public boolean handles(final int status, final MultivaluedMap<String, Object> headers) {
-        return Response.Status.BAD_REQUEST.getStatusCode() <= status && status < Response.Status.INTERNAL_SERVER_ERROR.getStatusCode();
+        return Response.Status.BAD_REQUEST.getStatusCode() < status && status < Response.Status.INTERNAL_SERVER_ERROR.getStatusCode();
```

Comment:
ah, so without this change 400's were handled as being 'fouten' instead of 'validatie fouten' so to say?

I think these mappers could do with a little JavaDoc explaining exactly what is going on? `Maps all HTTP responses with status code between 400 and 500 to a FoutException`?

Reply:
Done, could you have a look?

Reply:
Looks good!